TeamCode/src/main/java/suitbots/Robot.java

```diff
@@ -62,4 +64,13 @@ public boolean isAboveWhiteLine() {
             return false;
         }
     }
+
+    public boolean isGyroCalibrated() {
+        if(gyrochecker == 1) {
```

Comment:
`imu. isGyroCalibrated()` instead of the variable here. It takes a while for the gyro to calibrate after you call `initialize()`. You won't know it's done until `isGyroCalibrated()` returns true, so the `gyrochecker` variable is wrong at first.